src/components/header/share-modal/renderer.tsx

```diff
@@ -303,6 +301,20 @@ class ShareModal extends React.PureComponent<Props, State> {
             <h3>Update an existing Gist</h3>
             <p>To update an existing Gist, select it in the list and then click the button below to confirm.</p>
             <GistSelectWidget selectGist={this.selectGist.bind(this)} />
+            {this.props.isAuthenticated && (
+              <div className="share-input-container">
+                <label>
+                  File name: (optional)
+                  <input
+                    value={this.state.gistFileNameNew}
+                    name="gistFileNameNew"
+                    onChange={this.textChange.bind(this)}
+                    placeholder="Fill this only if you want to create a new file in the selected gist"
```

Comment:
That's not what placeholders are for. Let's make this a text description below. The placeholder should be the original file name (if any).